continuous-integration/linux/lint.sh

```diff
@@ -29,15 +29,19 @@ if [ $errorsFound -ne 0 ]; then
     exit 1
 fi
 
+source /etc/os-release || exit $?
+if [[ "$VERSION_ID" == "20.04" ]]; then
```

Comment:
Same here, do we even pass on any OS? I would think this script only makes sense to run on Ubuntu 20.04?

Reply:
This check is not required with -10 added to clang-tidy, which makes sense. It also makes sense that we only lint once. At one point I linted on both 18.04 and 20.04... will clean up.